firmware/lucidgloves-firmware/AdvancedConfig.h

```diff
@@ -27,24 +34,35 @@
 #endif
 
 
+#define MULTIPLEXER_DELAY   5 //How many microseconds should be delayed between multiplexer reads
+
+
 //ANALOG_MAX OVERRIDE:
-//uncomment and set as needed (only touch if you know what you are doing)
-//#define ANALOG_MAX 4095 
+//#define ANALOG_MAX 4095 //uncomment and set as needed
```

Comment:
I think this can be kept as it was? (unless it's another problem with an accidental diff)